feathr_project/feathr/api/app/routes/project.py

```diff
@@ -0,0 +1,36 @@
+
+from __future__ import annotations
+from typing import Optional
+
+from fastapi import APIRouter, HTTPException, Response,status
+from app.backends.purview_backend.purview_backend import PurviewBackend
+
+from app.core.configs import *
+from app.core.error_handling import verifyCode
+from app.models.features import Features
+
+router = APIRouter()
+backend = PurviewBackend()
+
+@router.get("/projects",tags=["Projects"])
+def get_all_projects(code : str, response: Response):
```

Comment:
is the convention to just use methods and not classes?

Reply:
Yes, ref : https://fastapi.tiangolo.com/tutorial/bigger-applications/#import-apirouter

Restful interfaces by nature should be stateless, and I think method-based is reasonable.